src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -358,13 +358,24 @@ bool SetGlobalPropertiesRequest::PrepareUIRequestDefaultVRHelpData(
         (*command_it->second)[strings::vr_commands][0];
   }
 
-  app->set_vr_help_title(smart_objects::SmartObject(app->name()));
-
-  out_params[strings::vr_help_title] = (*app->vr_help_title());
-  if (vr_help_items.length() > 0) {
+  if (!vr_help_items.empty()) {
     app->set_vr_help(vr_help_items);
-    out_params[strings::vr_help] = (*app->vr_help());
+  } else {
+    // No one AddCommand happened before, so get vrHelp from first vrSynonym
+    // that was taken at registration
+    LOG4CXX_DEBUG(logger_, "Create vrHelp from vrSynonyms");
+    if (!app->vr_synonyms()->empty()) {
```

Comment:
@LuxoftAKutsan it seems that vrSynonyms is an optional parameter at app registration, so it can be NULL. Nice catch! Fixed with 1af6cea.